internal/globaldefault/default.go

```diff
@@ -82,3 +82,31 @@ func SetupDefaultActivation(subshell subshell.SubShell, cfg DefaultConfigurer, r
 
 	return nil
 }
+
+func ResetDefaultActivation(subshell subshell.SubShell, cfg DefaultConfigurer) (bool, error) {
+	logging.Debug("Resetting globaldefault")
+
+	projectDir := cfg.GetString(constants.GlobalDefaultPrefname)
+	if projectDir == "" {
+		return false, nil // nothing to reset
```

Comment:
We may want to log this. Since we're relying on the config there is a way for users to have a default installation and be unable to remove it because of `state clean config`